### DISCUSSION TITLE
Question about Ignoring Torrents Younger Than...

### DISCUSSION THREAD

**Discussion by devanteweary** [Feramance/qBitrr#135] [Q&A] [answered]
Hey there,

So I notice that qBitrr will see a stalled torrent and research the media no matter how young that torrent is. 
I see a `IgnoreTorrentsYoungerThan` setting both under `[Settings]` and the `[Radarr]` and `[Sonarr]` sections.

I set these to an hour (3600) but they don't seem to do anything. 
The description in the config file states the one under `[Settings]` is only used if you use the recheck qbittorrent category.

What about those other two (radarr and sonarr sections)?

If not, I'll probably throw up a feature request for an overall IgnoreTorrentsYoungerThan-type setting.

**Comment by Feramance** [comment, marked as answer]
I'll adjust this with the next update, currently the logic goes (when enabled) `if stalled => search`
